play/type-shooter/js/app.js

```diff
@@ -34,18 +47,66 @@ const STARTING_LIVES = 10;
 (function initUI() {
     // show instructions modal
     $("#myModal").modal("show");
+
+    // keep page scrolled to the very top
+    document.querySelector("body").scrollTop = 0;
```

Comment:
You can actually just do ```document.body``` https://developer.mozilla.org/en-US/docs/Web/API/Document/body.